CONTRIBUTING.md

```diff
@@ -20,51 +20,47 @@ To get started, we need to have the following software installed:
 * [Kubebuilder Prerequisites](https://book.kubebuilder.io/quick-start.html#prerequisites) (go, docker, kubectl, kubebuilder, controller-gen)
 * [helm](https://helm.sh/docs/intro/quickstart/)
 
-# Deploy the controller locally
+# Developing Locally
 
-## Quick start
+## Quick start with Minikube
 
 Once you have installed the above requirements, run the following commands:
 
 * start minikube with containerd engine
   * `make minikube-start`
 * deploy cert-manager
   * `kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.3.0/cert-manager.yaml`
-* build the new image of the controller with your local changes
+* build the new image of the controller with your local files
   * `make minikube-build`
 * deploy the CRD and the controller on the minikube cluster
   * `make install`
 
-If the controller is already deployed and you want to test a new version of it, please run the `make restart` command.
+## Deploying Local Changes to Minikube
 
-The [samples](examples/) contains sample data which can be used to test your changes.
-
-* [demo.yaml](examples/demo.yaml) contains a bunch of testing resources you can apply directly to your cluster
-  * `kubectl create ns chaos-engineering`
-  * `kubectl -n chaos-engineering apply -f examples/demo.yaml`
-
-## Helpers scripts
+To deploy a new version of your local controller code when a version is already deployed, run:
+* `make minikube-build`
+* `make install`
+* `make restart`
 
-For verification on minikube we created some helper [scripts](scripts/):
+To deploy a new version of the CRD by modifying your local `api/v1beta1/disruption_types.go` (or a particular Subspec by modifying `api/v1beta1/disruption_types.go`), run:
+* `make install`
+* `make restart`
 
-* List pod interfaces:
-  * `./scripts/list_links.sh <pod_name>`
-* List traffic control filters of the given pod:
-  * `./scripts/list_tc_filters.sh <pod_name>`
-* List traffic control qdiscs of the given pod:
-  * `./scripts/list_tc_qdiscs.sh <pod_name>`
+## Testing Local Changes in Minikube
 
-## Minikube ISO
+The [samples](examples/) contains sample data which can be used to test your changes.
 
-See [minikube_image.md](docs/minikube_image.md)
+[demo.yaml](examples/demo.yaml) contains testing resources you can apply directly to your cluster in whatever namespace you choose (it can be `chaos-engineering`) by running:
+  * `kubectl create ns chaos-engineering`
```

Comment:
This should not be useful anymore since the namespace is created by the chart now